contract/models/contract_modification.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ContractModification(models.Model):
+
+    _name = 'contract.modification'
+    _description = 'Contract Modification'
+    _order = 'date asc'
+
+    date = fields.Date(
+        required=True,
+        string='Date'
+    )
+    description = fields.Text(
+        required=True,
+        string='Description'
+    )
+    contract_id = fields.Many2one(
+        string='Contract',
+        comodel_name='contract.contract',
+        required=True,
+        ondelete='cascade',
+        index=True
+    )
+    mail_notify = fields.Boolean(
```

Comment:
Be more precise in the field name. This may mean the contrary: that the notification should be done. Maybe you wanted to say `mail_notified`. Get use to the participle form of the verbs in these cases. Anyway, I think it's better to use `sent` as field name.